instructure/canvas-ios
# Dont show muted when new gradebook enabled

refs: MBL-13421
affects: Teacher
release note: Fixed an issue with muted assignments

Test Plan:
 - Turn off new gradebook for a course
 - create an assignment
 - Go into the gradebook and mute the assignment
 - Open the submission list and see that the
   header says muted
 - Turn on new gradebook for the course
 - Navigate to the submission list again (feature
   flags are refreshed on mount)
 - It should not say muted in the header
Note that after flipping the switch for the account
for new gradebook, it may take a few  minutes before
the course has the updated flag setting
